app/views/includes/head.pug

```diff
@@ -23,10 +23,13 @@ head
   meta(property='og:site_name', content='Cards for Humanity')
   meta(property='fb:admins', content='APP_ADMIN')
 
-  link(rel='stylesheet', href='/lib/bootstrap/css/bootstrap.css')
-  link(rel='stylesheet', href='/css/common.css')
-  link(rel='stylesheet', href='/css/animate.css')
-  link(rel='stylesheet', href='/css/style.css')
+  link(rel='stylesheet', href='/lib/bootstrap/dist/css/bootstrap.css')
+  link(rel='stylesheet', href='bundle/main-52c0965d31.css')
+  
+  //- link(rel='stylesheet', href='/css/common.css')
```

Comment:
So, since we are bundling all css to a single file (main.css), we don't need the commented out code here.